index.js

```diff
@@ -55,14 +55,20 @@ export class LinkIndexer {
    * Decode the block and index any CIDs it links to
    * @param {import('@ipld/car/api').Block} block
    * @param {object} [opts]
-   * @param {import('./decode.js').BlockDecoders} [opts.codecs]
+   * @param {import('./decode.js').BlockDecoders} [opts.codecs] - bring your own codecs
+   * @param {boolean} [opts.identityCidLink] - skip block index count if were processing an identity link cid
    */
   decodeAndIndex ({ cid, bytes }, opts) {
     const block = maybeDecode({ cid, bytes }, opts)
     if (!block) {
       this.undecodable++
     } else {
       this._index(block)
+      if (opts?.identityCidLink) {
```

Comment:
I'd not expose this option publically - you never want to set it as an API consumer.